.gitpod.yml

```
@@ -1,21 +1,19 @@
-# Using an updated Gitpod docker image (when using a non default JDK or tools)
-image:
-  file: .gitpod.Dockerfile
-
 tasks:
-  - init: sdk default java  11.0.15-tem && mvn clean verify
+  - name: Preview
+    command: gp ports await 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
+
+  - name: Setup and Build
+    before: |
+      # Download and Install Java 17.0.6-tem
+      yes Y | sdk install java 17.0.6-tem
+
+    init: |
+      mvn hpi:run -Dhost=0.0.0.0      
```

I don't understand why the init action runs the Jenkins controller and loads the plugin.  I think that a Gitpod environment is intended to be ready for a developer to do development work like adding tests or creating new features.  It seems premature to start the controller from `init`.

Can you explain why you chose to start the Jenkins controller from the init action?

If this is intentional, shouldn't the name be "Run Jenkins" rather than "Setup and Build"?
```suggestion
      mvn clean verify
```

---

This was my mistake. I have updated the `.gitpod.yml` file to install java on `init` as This task is expected to terminate. If it fails, the `command` property will not be executed.

the `command` property is executed last on every start and doesn’t have to terminate.

I have also changed the name to "Run Jenkins"

Thanks!